drivers/char/rust_example.rs

```diff
@@ -51,6 +51,8 @@ struct RustFile;
 impl FileOperations for RustFile {
     type Wrapper = Box<Self>;
 
+    kernel::in_file_operations!();
```

Comment:
I think a better name would be `declare_file_operations` or something.

Reply:
Sure, changed it.